core/src/test/java/com/amplifyframework/core/configuration/AmplifyOutputsDataTest.kt

```diff
@@ -253,6 +253,44 @@ class AmplifyOutputsDataTest {
         outputs.storage?.run {
             awsRegion shouldBe "us-east-1"
             bucketName shouldBe "myBucket"
+            buckets.size shouldBe 0
+        }
+    }
+
+    @Test
+    fun `parses multi-bucket storage configuration`() {
+        val json = createJson(
+            Keys.storage to mapOf(
+                Keys.region to "us-east-1",
+                Keys.bucket to "myBucket",
+                Keys.buckets to listOf(
+                    mapOf(
+                        Keys.region to "us-east-1",
+                        Keys.bucket to "myBucket",
+                        Keys.name to "name1"
+                    ),
+                    mapOf(
+                        Keys.region to "us-east-2",
+                        Keys.bucket to "myBucket2",
+                        Keys.name to "name2"
+                    )
+                )
+            )
+        )
+
+        val outputs = AmplifyOutputsData.deserialize(json)
+
+        outputs.storage.shouldNotBeNull()
+        outputs.storage?.run {
+            awsRegion shouldBe "us-east-1"
+            bucketName shouldBe "myBucket"
+            buckets.size shouldBe 2
+            buckets[0].name shouldBe "name1"
```

Comment:
suggest 

```
buckets[0].apply {
  name should be "myBucket"
  ...
}
```

Another option wouild be to just constructed an expectedList and do a shouldBe comparison on that.

Reply:
good suggestion, unit test updated.